### DISCUSSION TITLE
Jyers Firmware & Creality Smart Kit - will it work?

### DISCUSSION THREAD

**Discussion by DasSmeagle** [Jyers/Marlin#1337] [General Discussion]
Hi,

I am getting the Creality Smart Kit... Do I need to expect problems with Jyers Firmware? Anything special to set up when compiling Firmware?

https://de.aliexpress.com/item/1005002868833131.html?spm=a2g0s.8937460.0.0.56de2e0efiVfRM

I see, many people have problems getting printers to connect, even "plain Marlin" sometimes won't connect? Anything to make sure to get clear of problems?

Thanks,
- Oliver

**Comment by Cal-LF** [comment]
The Creality wifi box is known to have problems. It is not a reliable piece of hardware.

**Reply by DasSmeagle** [reply]
Yeah, that's why I am asking ;) How to avoid those problems? For some people it works fine. And it is really cheap, so I want to try it.

**Reply by DasSmeagle** [reply]
It doesn't work :( It does not connect to the printer, no reaction.

Any idea what I could try?

**Reply by DasSmeagle** [reply]
After the latest App Update everything works. But that thing is pretty useless. Can't even do a full screen video, only very small video in app.
